evm/src/executor/inspector/cheatcodes/fork.rs

```diff
@@ -140,6 +143,59 @@ pub fn apply<DB: DatabaseExt>(
             )
             .map(empty)
             .map_err(Into::into),
+        HEVMCalls::GetLogs(inner) => {
```

Comment:
We can extract this body into a function—it'll be easier to maintain down the road